src/components/Directory.vue

```diff
@@ -96,6 +96,13 @@ export default {
   margin: 0;
 }
 
+.directory__phone-number {
```

Comment:
I have doubts about this styling, it might be useful for the end user to know somehow that the phones are clickable. Also, on mobile it might be necessary to have extra space, to make sure users can tap the right option. Please check with Keren, maybe we can improve usability a bit here.

Reply:
I already talked with Lu, and she told me how to do it. The styles are validated.